starter-code/bin/seeds.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+const  celebModel= require('../models/celebrity');
+const movieModel = require('../models/movie');
+
+mongoose
+  .connect('mongodb://localhost/starter-code', {useNewUrlParser: true, useUnifiedTopology: true})
+  .then(x => {
+    console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
+  })
+  .catch(err => {
+    console.error('Error connecting to mongo', err)
+  });
+
+const Celebrities=[
+    {name: "Homer Simpson", occupation: "Nuclear plant operator", catchPhrase: `D'oh!`},
+    {name: "Leroy Jenkins", occupation: "AFK", catchPhrase: `LEROOOOY JENKINS`},
+    {name: "Julius Caesar", occupation: "Politician", catchPhrase: `Alea Jacta Est`}
+]
+
+celebModel.insertMany(Celebrities)
+    .then(()=> console.log(`Database updated.`))
+        .then(()=>{
```

Comment:
you don't need a second .then in this case. Remember .then and .catch are only used on promises. The previous line is a simple console.log, so you can use that line and then simply in the next line, close the mongoose connection.